.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/base",
+  "customizations": {
+    "vscode": {
+      "extensions": ["lizebang.bash-extension-pack"]

```

Comment:
Could you add the `EditorConfig.EditorConfig` extension?

Reply:
fixed in https://github.com/spenserblack/actions-wiki/pull/36/commits/c1b57e2a290873c6b50caffc95e525d4ad703fb7